src/components/pages/ReportForm/index.vue

```diff
@@ -0,0 +1,87 @@
+<template lang="pug">
+  .issue
+    .container(v-if="project")
+      h2 {{ $t('title', { project: project.name }) }}
+
+      form-wizard(@on-complete='onComplete', 
+                  color='#00A1DF',
+                  title="", subtitle="",
+                  :back-button-text="$t('buttons.back')",
+                  :next-button-text="$t('buttons.next')",
+                  :finish-button-text="$t('buttons.finish')")
+
+        tab-content(:title="$t('step1.title')", :before-change="validateStep1")
+          step1
+
+        tab-content(:title="$t('step2.title')", :before-change="validateStep2")
+          step2
+
+        tab-content(:title="$t('step3.title')")
+          step3
+
+    loader(v-else)
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+import reportService from '@/api/report'
```

Comment:
Does this relatives imports with the `@` actually work? That's pretty awesome! :+1: